libs/src/services/ethContracts/index.js

```diff
@@ -176,6 +176,18 @@ class EthContracts {
     )
   }
 
+  /**
+   * Determines whether the major and minor versions are equal
+   * @param {string} version string 1
+   * @param {string} version string 2
+   */
+  hasSameMajorAndMinorVersion (version1, version2) {
```

Comment:
are we going to remove the isValidSPVersion method?

Reply:
Yes, but not in this PR

Reply:
It's still used by other code that needs to be removed more carefully